app/views/facilities/_facility_fields.html.haml

```diff
@@ -10,7 +10,16 @@
     %label.legend= f.label :banner_notice
     = f.input :banner_notice, input_html: { class: "editor__simple" }, hint: text(".hints.banner_notice"), label: false
 
-%fieldset.well
+- if SettingsHelper.feature_on?(:facility_tile_list)
+  .well
+    %label= text("facilities.thumbnail.thumbnail")
+    = image_tag (@facility.thumbnail.url), class: 'thumbnail_preview'
```

Comment:
I think we need a `if @facility.thumbnail?` conditional around this and the remove input. I'm seeing a missing /broken image when there isn't a file present.
![screen shot 2018-12-17 at 3 52 24 pm](https://user-images.githubusercontent.com/1099111/50117912-d2749280-0213-11e9-89d6-c97ca108ef16.png)